public/javascripts/controllers/search-ctrl.js

```diff
@@ -107,6 +107,24 @@ angular.module('jasmic.controllers')
                 }
             };
 
+
+            $scope.highlightSearchItem = function($event) {
```

Comment:
@nick22891 try not to mix view logic with controller logic.  The view can be entirely manipulated in by using angular's framework, leaving these files free from style renderings, and DOM object manipulation.  Please review.